bessctl/module_tests/worker_split.py

```diff
@@ -62,6 +62,52 @@ def test_worker_split(self):
 
             bess.reset_all()
 
+    def test_worker_split_mask_mode(self):
+        NUM_WORKERS = 6
+
+        masks = dict()
+        # map even workers to gate 0
+        masks[0] = reduce(lambda x,y: x|y, [0x1 << 2*i for i in range(NUM_WORKERS/2)])
```

Comment:
Need `NUM_WORKERS // 2` to force integer divide in python3:
```
>>> range(6/2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'float' object cannot be interpreted as an integer
```

Reply:
Great catch! Fixed